src/components/Detail.jsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+// import { useSelector } from 'react-redux/es/hooks/useSelector';
```

Comment:
- [ ] Kindly remove commented code from this file as well. Remember, Making console statements public is not a good idea at all, it may bring your app at a security risk. Please remove the commented code for a cleaner code base.

Reply:
done